src/main/java/com/firecontroller1847/truediscordlink/commands/CommandUnlink.java

```diff
@@ -76,6 +77,19 @@ public boolean onCommand(@NotNull CommandSender sender, @NotNull Command command
 
             // Unlink
             player.sendMessage(plugin.getTranslation("linking.unlink"));
+
+            // Check if enabled
+            if (!plugin.getConfig().getBoolean("bot.linking.notify_linkage.unlink.notify")) {
+                return true;
+            }
+
+            String channelId = plugin.getConfig().getBoolean("bot.linking.notify_linkage.use_separate") ? plugin.getConfig().getString("bot.linking.notify_linkage.unlink.channel") : plugin.getConfig().getString("bot.linking.notify_linkage.link.channel");
+            TextChannel channel = ((TrueDiscordLink) plugin).getDiscordManager().getApi().getChannelById(channelId).orElseThrow(() -> new Exception("Unlink Notification Channel cannot be null!")).asTextChannel().orElseThrow(() -> new Exception("Unlink Notification Channel must be a text channel"));
+
+            channel.sendMessage(plugin.getTranslation("linking.discord.notify_linkage.unlink",
+                    new String[] { "%username%", player.getName() },
+                    new String[] { "%tag%", plugin.getConfig().getBoolean("bot.linking.notify_linkage.unlink.ping") ? "<@" + userId + ">" : ((TrueDiscordLink) plugin).getDiscordManager().getApi().getUserById(userId).get().getDiscriminatedName() }
```

Comment:
I'm not sure I entirely enjoy this concept of a "ping" configuration option. Maybe we could add an additional, optional argument called "%mention%" alongside "%tag%" which would perform the same functionality without the need for an additional configuration option check